### PR TITLE
iexec-worker PR#602 repatriation

### PR DESCRIPTION
**Pull request by Natchica**

### CODE REVIEW

File: src/test/resources/wiremock/mappings/config-server.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "url": "/config/chain"
+  },
+  "response": {
+    "status": 200,
+    "body": "{\"chainId\":255,\"chainNodeUrl\":\"http://localhost:8545\",\"blockTime\":\"PT5s\",\"iexecHubContractAddress\":\"0xC129e7917b7c7DeDfAa5Fff1FB18d5D7050fE8ca\",\"sidechain\":\"true\"}",
+    "headers": {
+      "Content-Type": "application/json"
+    }
+  }
+}
```
Comment (jbern0rd) [inline src/test/resources/wiremock/mappings/config-server.json]: missing new line at end of file

File: src/test/resources/wiremock/mappings/worker-config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "status": 200,
+    "body": "{\"workerPoolAddress\":\"0x1\",\"schedulerPublicAddress\":\"0x2\",\"blockchainAdapterUrl\":\"http://localhost:{{request.port}}\",\"configServerUrl\":\"http://localhost:{{request.port}}\",\"resultRepositoryURL\":\"http://localhost:{{request.port}}\",\"askForReplicatePeriod\":\"5\",\"requiredWorkerVersion\":\"x.y.z\"}",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "transformers": [
+      "response-template"
+    ]
+  }
+}
```
Comment (jbern0rd) [inline src/test/resources/wiremock/mappings/worker-config.json]: missing new line at end of file

Comment (jbern0rd) [reply src/test/resources/wiremock/mappings/worker-config.json]: That's the flag meaning

File: src/test/java/com/iexec/standalone/chain/EnclaveAuthorizationServiceTests.java [hunk 3]
```diff
@@ -115,6 +112,6 @@ void isNotVerifiedEnclaveSignatureSinceWrongEnclaveSignature() throws InvalidAlg
         Credentials credentials = Credentials.create(Keys.createEcKeyPair());
 
         boolean isVerifiedEnclaveSignature = enclaveAuthorizationService.isVerifiedEnclaveSignature(chainTaskId, resultHash, resultSeal, "0x1", credentials.getAddress());
-        Assertions.assertFalse(isVerifiedEnclaveSignature);
+        assertThat(isVerifiedEnclaveSignature).isFalse();
     }
 }
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/chain/EnclaveAuthorizationServiceTests.java]: The missing line at end of file was fixed on worker side in the mentioned PR

**Review by jbern0rd — APPROVED** [review]

**Review by thewhitewizard — APPROVED** [review]

**Pull request merged**